tools/reviewer/job/sync/full_github_pr.proto

```diff
@@ -5,55 +5,37 @@ package com.google.startupos.tools.reviewer.job.sync;
 option java_package = "com.google.startupos.tools.reviewer.job.sync";
 option java_outer_classname = "GithubPullRequestProtos";
 
+// TODO: Remove unused fields
 message PullRequest {
 
   int64 id = 1;
   int64 number = 2;
+  // valid values: `open`, `closed`
   string state = 3;
   string title = 4;
   // Pull Request author
   User user = 5;
   string body = 6;
+  // expected format for created_at and updated_at fields:
```

Comment:
`s/expected/Expected`, and how about we do this comment in 1 line? Will it fit?

Reply:
The Formatter splits this into two lines.